frameworks/Java/servicetalk/src/main/java/servicetalk/Server.java

```diff
@@ -0,0 +1,100 @@
+
+
+
+public final class Server {
+
+
+    public static void main(String[] args) throws Exception {
+
+        /**
+         * Disable  AsyncContext
+         */
+        AsyncContext.disable();
+
+        /**
+        *   Factory to implement io pooling
+        */
+        IoExecutor ioExecutor = NettyIoExecutors.createIoExecutor(
+          Runtime.getRuntime().availableProcessors(),
+          new IoThreadFactory("io-pool")
+        );
+
+        /**
+         * Factory to disable headers validation
+         */
+
+        DefaultHttpHeadersFactory headersFactory = new DefaultHttpHeadersFactory(false,false);
+
+        try{
+            HttpSerializationProvider serializer = HttpSerializationProviders.jsonSerializer(new JacksonSerializationProvider());
+//            Create a custom server builder with performance enhancements
+            HttpServers
+              .forPort(8080)
+              .executionStrategy(HttpExecutionStrategies.noOffloadsStrategy())
+              .ioExecutor(ioExecutor)
+              .disableDrainingRequestPayloadBody()
+              .protocols(HttpProtocolConfigs.h1().headersFactory(headersFactory).build())
+              .appendConnectionAcceptorFilter(delegate -> new ConnectionAcceptor() {
+                  @Override
+                  public Completable accept(ConnectionContext context) {
+                      ((NettyConnectionContext)context).updateFlushStrategy((current, isOrig) -> FlushStrategies.flushOnEnd());
+                      return delegate.accept(context);
+                  }
+              })
+              .listenAndAwait((ctx, request, responseFactory) -> {
+                  ((NettyConnectionContext)ctx).updateFlushStrategy(((current, isCurrentOriginal) -> FlushStrategies.flushOnEach()));
+                  if(request.path().equals("/json")) {
+                      Map<String, String> obj = new HashMap<String, String>();
+                      obj.put("message", "Hello, World!");
+                      return succeeded(responseFactory.ok()
+                        .payloadBody(obj, serializer.serializerFor(Map.class))
+                        .addHeader("Date", GetCurrentTime())
+                        .addHeader("Server", "TFB"));
+                  }
+
+                  if(request.path().equals("/plaintext")) {
+                      return succeeded(responseFactory.ok()
+                        .payloadBody("Hello, World!", HttpSerializationProviders.textSerializer())
+                        .addHeader("Date", GetCurrentTime())
+                        .addHeader("Server", "TFB"));
+                    };
+                    return null;
+                })
+                .awaitShutdown();
+        } catch (Exception e) {
+//            logging is not allowed
+        }
+    }
+
+    public static String GetCurrentTime() {
+        SimpleDateFormat formatter = new SimpleDateFormat("EE, dd MMM yyyy kk:mm:ss z  ");
+        Date date = new Date();
```

Comment:
`Date` and `SimpleDateFormat` are part of the legacy Java time libraries, which were effectively replaced by the new `java.time` libraries in Java 8.  Here, what you're looking for is `DateTimeFormatter`.  Other frameworks show how to use it.  Example:  https://github.com/TechEmpower/FrameworkBenchmarks/blob/b843c9ea4f9096435ae52bf99b0182737a2c1edc/frameworks/Java/armeria/src/main/java/hello/App.java#L25-L27

Note that `DateTimeFormatter` is immutable, so a single instance can be reused every time you want to format a date.  And luckily there is already a built-in `DateTimeFormatter` constant that has the format you want here.  One of `SimpleDateFormat`'s problems is that it is mutable -- formatting a date modifies the `SimpleDateFormat` instance -- so you have to construct a new instance for each date like this.